CMakeLists.txt

```diff
@@ -49,11 +49,11 @@ endif()
 
 set(CPACK_PACKAGE_VERSION_MAJOR "24")
 set(CPACK_PACKAGE_VERSION_MINOR "0")
-set(CPACK_PACKAGE_VERSION_PATCH "0")
```

Comment:
Why are we going to 24.0.1 instead of 24.0.0?
It I not a problem but I wonder why.

Reply:
I can answer my own question. Because that number is specifically for marking release candidates.

Reply:
Yeah, it also matches the RC number, so 0 is not the first version.